.github/workflows/z_fastai.yml

```diff
@@ -0,0 +1,42 @@
+name: z_fastai
```

Comment:
If I recall correctly, fastaiv2 example didn't work as reported in https://github.com/optuna/optuna/issues/2107.
But now it seems working.

If this observation is correct, could you close the issue mentioned above once this pull request lands?